test/src/commands/create_test.dart

```diff
@@ -484,7 +484,7 @@ void main() {
             ),
           ).thenAnswer((_) async {
             File(p.join('.tmp', 'pubspec.yaml')).writeAsStringSync(pubspec);
-            return 62;
+            return List.filled(62, const GeneratedFile.created(path: ''));
```

Comment:
Nit: this is being repeated on the test, maybe would be worth adding it to a method for the sake of DRY

Reply:
Yeah I'm fine either way the reason I left it as is is because I don't mind repeating code in tests in order to keep the tests self-contained/easy to understand. Happy to adjust if you prefer we create a helper for this. Let me know what you had in mind 👍

Reply:
I was thinking on a generateFileList which just receive the length, so some details of how the file is mocked is not repeated.

Either way is just a nit, I know DRY in tests is a complicated matter hahahha

Reply:
I updated it a bit, lmk what you think

Reply:
LGTM! 🚀